### PR TITLE
Bug 1130632 - Add FirefoxOS version information to gaia build

### PR DESCRIPTION
**Pull request by rickychien**
https://bugzilla.mozilla.org/show_bug.cgi?id=1130632

### CODE REVIEW

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b7d6760af3ee)

File: build/config/common-settings.json [hunk 1]
```diff
@@ -119,6 +119,7 @@
    "lockscreen.locked": true,
    "lockscreen.unlock-sound.enabled": false,
    "message.sent-sound.enabled": true,
+   "moz.b2g.version": "3.0.0.0-prerelease",
```
Comment (KevinGrandon) [inline build/config/common-settings.json]: Maybe we should just do ```3.0``` here?

Comment (zbraniecki) [inline build/config/common-settings.json]: I don't fully understand why we even have 3.0.0.0-prerelease if the versions we use and display to the user are 2.1, 2.2, 2.3 (and 2.3m, 2.3s etc.)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=669bc2a4e1ee)

**Comment by try-server-hook** [comment]
[RickyChien Ricky Chien (RickyChien)](https://github.com/RickyChien) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=51d3bda80d32)

**Pull request merged**